### ISSUE TITLE
Error when saving records with relationships

### ISSUE THREAD

**Issue by marlongichie** [Vinelab/NeoEloquent#56]
Individual records are saved as expected. However, when saving a relationship, I get a `[InvalidArgumentException] Database [1] not configured.' error.

Here is some info from the logs:

```
'InvalidArgumentException' with message 'Database [1] not configured.' in /home/vagrant/code/antullia-beta/vendor/laravel/framework/src/Illuminate/Database/DatabaseManager.php:248
```

Database configuration looks like below, and I have set `'default' => 'neo4j',`

```
'neo4j' => [
            'driver' => 'neo4j',
            'host' => env('DB_HOST', 'localhost'),
            'port' => '7474',
            'username' => null,
            'password' => null
        ]
```

Anyone else experiencing this?

**Comment by Mulkave** [comment]
Should have been fixed in 1dcfbf5364d9bdb4377e301b21ea89988f4e2137 please try a composer update and make sure you have version 1.2.4 installed.

**Comment by Mulkave** [comment]
@marlongichie any updates on this?
